app/src/main/java/org/exoplatform/activity/AddDomainServerActivity.java

```diff
@@ -7,6 +7,9 @@
 import android.content.Intent;
 import android.os.Build;
 import android.os.Bundle;
+import android.text.Editable;
+import android.text.InputFilter;
+import android.text.TextWatcher;
```

Comment:
This change is not needed

Reply:
@wbenabdessalem could you remove the imports above ?

Reply:
Ok